src/main/java/com/book/model/JwtModel.java

```diff
@@ -0,0 +1,17 @@
+package com.book.model;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
```

Comment:
The best Lombok annotation for POJO is @Data: https://projectlombok.org/features/Data